airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/manifest.yaml

```diff
@@ -600,6 +600,80 @@ definitions:
       - type: CustomTransformation
         class_name: "source_tiktok_marketing.components.transformations.TransformEmptyMetrics"
 
+  ads_reports_by_country_daily_stream:
+    type: DeclarativeStream
+    name: ads_reports_by_country_daily
+    $parameters:
+      path: "report/integrated/get/"
+      data_level: "AUCTION_AD"
+      filtering:
+        [
+          {
+            "field_name": "ad_status",
+            "filter_type": "IN",
+            "filter_value": "[\\\"STATUS_ALL\\\"]",
+          },
+        ]
+      report_metrics:
+        [
+          "campaign_name",
+          "campaign_id",
+          "adgroup_name",
+          "placement_type",
+          "tt_app_id",
+          "tt_app_name",
+          "mobile_app_id",
+          "promotion_type",
+          "dpa_target_audience_type",
+          "conversion",
+          "cost_per_conversion",
+          "conversion_rate",
+          "real_time_conversion",
+          "real_time_cost_per_conversion",
+          "real_time_conversion_rate",
+          "result",
+          "cost_per_result",
+          "result_rate",
+          "real_time_result",
+          "real_time_cost_per_result",
+          "real_time_result_rate",
+          "secondary_goal_result",
+          "cost_per_secondary_goal_result",
+          "secondary_goal_result_rate",
+          "adgroup_id",
+          "ad_name",
+          "ad_text",
+          "total_purchase_value",
+          "total_onsite_shopping_value",
+          "onsite_shopping",
+          "vta_purchase",
+          "vta_conversion",
+          "cta_purchase",
+          "cta_conversion",
+          "total_pageview",
+          "complete_payment",
+          "value_per_complete_payment",
+          "total_complete_payment_rate",
```

Comment:
Some metrics were added recently [here](https://github.com/airbytehq/airbyte/pull/56451). Can we make sure we have them for the new streams as well?

Reply:
Added aside from `purchase` related metrics. According to https://business-api.tiktok.com/portal/docs?id=1751443967255553, they are not supported with 3 dimension grouping